utam-compiler/src/main/java/utam/compiler/helpers/BasicElementActionType.java

```diff
@@ -122,14 +122,6 @@
   }
 
   public static ActionType getActionType(String apply, TypeProvider elementType, String elementName) {
-    if (!BasicElementInterface.isBasicType(elementType)
-        && !elementType.isSameType(ROOT_ELEMENT_TYPE)) {
-      throw new UtamError(
-          String.format(
-              ActionableActionType.ERR_NOT_HTML_ELEMENT,
-              elementName,
-              elementType.getSimpleName()));
-    }
```

Comment:
removed this validation because we already made sure that element that gets basic action is basic earlier in the code